test/utils/index.ts

```diff
@@ -1,52 +1,40 @@
-import Lumino from "lumino-js-sdk";
+import Lumino from 'lumino-js-sdk';
+import { LuminoTestEnvironment } from '../../src/types/lumino-test-environment';
+import { LuminoNode } from '../../src/types/node';
+import { ChannelState } from '../common';
 
 export function isEmpty(value: any): boolean {
-  return Array.isArray(value) ? value.length <= 0 : !value;
+    return Array.isArray(value) ? value.length <= 0 : !value;
 }
 
 export function sleep(ms: number) {
-  return new Promise((resolve) => setTimeout(resolve, ms));
+    return new Promise((resolve) => setTimeout(resolve, ms));
 }
 
-export enum ChannelState {
-  OPEN = "opened",
-  CLOSED = "closed",
-}
-
-export class ChannelTestCase {
-  token: string; // hex address
-  partner: string; // hex address
-  deposit: number; // in Wei
-  balance: number; // in Wei
-  state: ChannelState; // "opened", "closed"
-
-  constructor(
-    tokenAddr: string,
-    partnerAddr: string,
-    totalDeposit: number,
-    balance: number,
-    channelState: ChannelState
-  ) {
-    this.token = tokenAddr;
-    this.partner = partnerAddr;
-    this.deposit = totalDeposit;
-    this.balance = balance;
-    this.state = channelState;
-  }
+export async function verifyEnv(env: LuminoTestEnvironment): Promise<void> {
+    expect(Object.keys(env.nodes).length).toBe(2);
+    const firstNode = env.nodes.node0 as LuminoNode;
+    const secondNode = env.nodes.node1 as LuminoNode;
+    const firstAddr = (await firstNode.client.sdk.getAddress()).our_address;
+    const secondAddr = (await secondNode.client.sdk.getAddress()).our_address;
+    expect(isEmpty(firstAddr)).toBe(false);
+    expect(isEmpty(secondAddr)).toBe(false);
```

Comment:
What if the env have more than two nodes? or less?

Reply:
I think when we create the validator for the topologies this will be solved and we will remove this function.

Reply:
agreed